src/Pages/registerTecnician.tsx

```diff
@@ -0,0 +1,93 @@
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import technicianSchema, {
+  registerTechnicianType,
+} from "../schemas/registerTechnician";
+import { zodResolver } from "@hookform/resolvers/zod";
+import useRegister from "../hook/useRegister";
+import "react-toastify/dist/ReactToastify.css";
+import Input from "../components/input/input";
+import logo from "../../assets/Logo.png";
+import { LuUser2 } from "react-icons/lu";
+import { MdOutlineMailOutline } from "react-icons/md";
+import { RiKey2Line, RiCellphoneLine } from "react-icons/ri";
+import ButtonPrimary from "../components/buttonPrimary/buttonPrimary";
+import SecondaryBackground from "../components/SecondaryBackground/SecondaryBackground";
+
+const RegisterTechnician = () => {
+  const { registerTechnician } = useRegister();
+
+  const methods = useForm<registerTechnicianType>({
+    resolver: zodResolver(technicianSchema),
+  });
+
+  const {
+    handleSubmit,
+    formState: { errors, isValid },
+  } = methods;
+
+  const onSubmit: SubmitHandler<registerTechnicianType> = (data) =>
+    registerTechnician(data);
```

Comment:
adicionar espaço depois das funções para ficar mais fácil de ler
```suggestion
    registerTechnician(data);
  
```